### ISSUE TITLE
List existing languages

### ISSUE THREAD

**Issue by hugobarragon** [meikidd/locale-code#12]
I have read the docs and i could not find and existing method in the api to list available languages any  , the closes i saw was `getLanguages` but it requires a list of languages is there anything i am not seeing?

**Comment by meikidd** [comment]
Yes, right now we don't have something called `getAllLanguages`, if you only care about Languages, I recommend this module [iso-639-1](https://github.com/meikidd/iso-639-1) which has the things you want. And we also can add `getAllLanguages` under this module if really needed.

**Comment by hugobarragon** [comment]
Yes, that would be great, for now I changed to ISO-639-1, but I think this library is the way to go, because languages are different in some countries and if we want to do something for people of that country, if we want to target them, we need to present something that their specific language, some examples: Portuguese the two big ones (PT-PT, PT-BR):in Portugal if something is presented in pt-br instead of PT-PT (many words are different) people get offended and on vice versa, probably in England the same they are presented pages/texts in English, that is en-us....  
And as i want to do something dynamic, the user is the one translating, i need the list of languages so they can select the ones they want. , think something that would list at least the codes, would be great, then from the codes with the rest of the api users of this library can get the rest...
 probably i can help you out, but only when i finish sometings.
